src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/UWPtoWinAppSDKUpgrade/WinUIPropertiesUpdater.cs

```diff
@@ -18,6 +21,12 @@ public class WinUIPropertiesUpdater : IUpdater<IProject>
     {
         public const string RuleID = "UA302";
 
+        private const string CsWinRTLogMessageFormat = "A CsWinRTIncludes property with value {0} has been added.\n" +
```

Comment:
just a note you can use literal strings (`@" "`) and have line returns right in the string itself instead of concatenation.